src/applications/ivc-champva/10-10d-extended/config/form.js

```diff
@@ -105,6 +114,41 @@ const formConfig = {
         },
       },
     },
+    sponsorInformation: {
+      title: 'Sponsor information',
+      pages: {
+        page6: {
+          path: 'sponsor-info',
+          title: 'Sponsor`s name and date of birth',
+          ...sponsorNameDobSchema,
+        },
+        page7: {
+          path: 'sponsor-identification-info',
+          title: `Sponsor's identification information`,
+          ...sponsorIdentificationSchema,
+        },
+        page8: {
+          path: 'sponsor-status',
+          title: 'Sponsor`s status',
+          ...sponsorStatus,
+        },
+        page9: {
+          path: 'sponsor-status-details',
+          title: 'Sponsor`s status details',
+          ...sponsorStatusDetails,
+        },
+        page10: {
+          path: 'sponsor-mailing-address',
+          title: 'Sponsor`s mailing address',
+          ...sponsorAddress,
+        },
+        page11: {
+          path: 'sponsor-contact-information',
+          title: 'Sponsor`s contact information',
+          ...sponsorContactInfo,
+        },
```

Comment:
Looks good! Just needs depends functions to make sure we ask the right questions depending on sponsor's status

```suggestion
        page8: {
          path: 'sponsor-status',
          title: 'Sponsor`s status',
          depends: formData => get('certifierRole', formData) !== 'sponsor',
          ...sponsorStatus,
        },
        page9: {
          path: 'sponsor-status-details',
          title: 'Sponsor`s status details',
          depends: formData =>
            get('certifierRole', formData) !== 'sponsor' &&
            get('sponsorIsDeceased', formData),
          ...sponsorStatusDetails,
        },
        page10: {
          path: 'sponsor-mailing-address',
          title: 'Sponsor`s mailing address',
          depends: formData => !get('sponsorIsDeceased', formData),
          ...sponsorAddress,
        },
        page11: {
          path: 'sponsor-contact-information',
          title: 'Sponsor`s contact information',
          depends: formData => !get('sponsorIsDeceased', formData),
          ...sponsorContactInfo,
        },
```